ibexa/commerce/4.0.x-dev/encore/ez.webpack.config.manager.js

```diff
@@ -1,33 +1,44 @@
 const fs = require('fs');
```

Comment:
Are we going to rename (ibexify 😉  ) this file?

Reply:
In the future - yes. ;) Scripts should accept both versions (`ez.xxx.js` and `ibexa.xxx.js`) but it requires backend change and is waiting for Sławek.